lively.ide/js/objecteditor/index.js

```diff
@@ -60,6 +60,7 @@ export class ObjectEditor extends Morph {
         evalEnvironment
       });
     }
+    win.doNotAcceptDropsForThisAndSubmorphs();
```

Comment:
Same as with the browser, what happens for instance with the importController? It will happily continue accepting drops. So either more calls of this are needed or we need to adjust the semantics of this method. (See last comment)